tests/argminmax_test.rs

```diff
@@ -44,6 +44,40 @@ fn test_argminmax_vec() {
     assert_eq!(max, ARRAY_LENGTH - 1);
 }
 
+
+#[test]
+fn test_argminmax_vec_with_nans() {
+    let mut data: Vec<f32> = (0..ARRAY_LENGTH).map(|x| x as f32).collect();
+    utils::insert_nans(&mut data, 0.05);
+    // Test owned vec
+    let (min, max) = data.argminmax();
+    assert_eq!(min, 0);
+    assert_eq!(max, ARRAY_LENGTH - 1);
+    // Test borrowed vec
+    let (min, max) = (&data).argminmax();
+    assert_eq!(min, 0);
+    assert_eq!(max, ARRAY_LENGTH - 1);
+}
+
+
+#[test]
+fn test_argminmax_vec_with_start_end_nans() {
+
+    let mut data: Vec<f32> = (0..100).map(|x| x as f32).collect();
+    data[0] = f32::NAN;
+    data[100-1] = f32::NAN;
```

Comment:
This is a deliberately adversarial test that fails.

Reply:
This test fails because now (0, 0) will be returned (as we have return NaNs as default behavior). To achieve the behavior as desired by this test you should cal `.nanargminmax` instead. This complies with numpy's behavior.

What do you think of this? I'm still unsure whether this is the best "default" behavior

Reply:
I agree with having to call a version that explicitly supports `NaN`s if your data has them.

In general, with NaNs, all bets are off, so I'm surprised (and extremely grateful) that we're adding this level of support to `argminmax`.